sql/o.karmanskyi_homework_25.sql

```diff
@@ -0,0 +1,25 @@
+SELECT *
+	FROM PersonsDB.dbo.Persons
+	WHERE Gender = 'M'
+
+SELECT *
+	FROM PersonsDB.dbo.Persons
+	WHERE Age > 18
+
+SELECT *
```

Comment:
that's not critical, but for null it's better to use IS operator
you can read it here why
https://stackoverflow.com/questions/9581745/sql-is-null-and-null
query will work same